src/main/java/com/dtolabs/rundeck/plugin/overthere/OTWinRMNodeExecutor.java

```diff
@@ -158,6 +165,10 @@ public OTWinRMNodeExecutor(final Framework framework) {
                     "If true, use HTTP instead of WSMAN for the SPN used for Kerberos Authentication.",
                     true, "false"))
 
+            .property(PropertyUtil.select(CONFIG_CMD_TYPE, "Command type",
+                    "Command type to use: CMD, Powershell",
```

Comment:
Instead of:
`"Command type to use: CMD, Powershell"`

Go for:
`String.format("Supported command types: %s, %s", CMD_TYPE_CMD, CMD_TYPE_POWERSHELL)`

This will avoid typo in the name of commands